vdbench.py

```diff
@@ -1,367 +1,367 @@
-# -*- coding: utf-8 -*-
-#pylint : disable=E0401
-"""
-@author: SUSHANT KEDAR				
-DATACORE SOFTWARE PVT LTD CONFIDENTIAL
-THIS SPEC IS THE PROPERTY OF DATACORE SOFTWARE PVT LTD.IT SHALL NOT BE
-COPIED, USED,TRANSLATED OR TRANSFERRED IN WHOLE OR IN PART TO ANY THIRD
-PARTY WITHOUT PRIOR WRITTEN PERMISSION OF DATACORE SOFTWARE PVT LTD.
-File Name	:	vdbench.py
-Description	:	This script used to execute vdbench tool and create
-                HTML reports.
-
-"""
-import os
-import sys
-from configparser import ConfigParser
-import subprocess
-path_ = os.path.abspath("../../../Interface/REST")
-sys.path.insert(0, path_)
-from ILDC import ILDC
-sys.path.insert(0, os.path.abspath("../../../Lib/VdBenchLib"))
-from error_log import LogCreat
-
-class VdBenchRun():
-    '''
-    Class:- VdBenchRun
-    This class executes vdbench 4k fill and VSI/VDI/ORACLE/SQL
-    wokload as user request.
-    Arguments : None
-    Return: None
-    '''
-    def __init__(self):
-        configur = ConfigParser()
-        configur.read(r"../../../Config/VdBench_config/VDBench_config.ini")
-        self.vdbench_path = configur.get('Vdbench run', 'vdbench_executable_path')
-        self.time_stamp = configur.get('first run', 'start')
-        self.build = ''
-        self.new_ = ''
-    def run(self,vd_name, workload, diskindex):
-        '''
-        This method execute workload of VdBench tool.
-        Arguments (str, str, int): vd_name, workload,diskindex
```

Comment:
In the comment section, you must define what each parameter (arguments) is used in the function. One line definition is must

Reply:
Its really hard to give that much documentation there so please skip this comment